red/runtime/nodes/flows/util.js

```
@@ -180,6 +183,155 @@ module.exports = {
+        
+        // Deep copy container structures to avoid mutation
+        for (var flowId in existingFlowConfig.flows) {
+            if (existingFlowConfig.flows.hasOwnProperty(flowId)) {
+                newConfig.flows[flowId] = {
+                    id: existingFlowConfig.flows[flowId].id,
+                    nodes: {},
+                    configs: {},
+                    subflows: {}
+                };
+                // Copy references (shallow)
+                for (var nodeId in existingFlowConfig.flows[flowId].nodes) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'nodeId'. Consider to delete it or rename it.

'nodeId' is defined here:


(lint/suspicious/noRedeclare)